pilot/pkg/config/kube/gateway/testdata/reference-policy-service.status.yaml.golden

```diff
@@ -64,9 +64,10 @@ status:
       status: "True"
       type: Accepted
     - lastTransitionTime: fake
-      message: All references resolved
-      reason: ResolvedRefs
-      status: "True"
+      message: backendRef httpbin/default not accessible to a route in namespace "istio-system"
```

Comment:
can we put positive an negative cases into different routes so we can ensure we have coverage of both? this may mask issues by merging them